docs/reqs/reqs.tex

```diff
@@ -192,118 +192,130 @@ \section{Goal}
+	\item The machine learning model's processing latency is one second or less.
 \end{enumerate}
 
 \subsubsection{Base Station}
-\begin{enumerate}[{F.BS}.1:]
+\begin{enumerate}[label=F.BS.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
     \item Ground station displays the video data overlaid with the ML model data.
 	\item ML model data is presented in the form of bounding boxes.
 \end{enumerate}
 
 \section{Non-Functional Requirements}\label{section:nonfuncrec}
 
-TODO: We might have to split non-functional requirements into nf-reqs and constraints (the capstone docs say they're separate, but PC's don't).
-
 \subsection{Multirotor RPAS}
-\begin{enumerate}[{NF.FP}.1:]
-    \item Drone has at least 50\% flight duration of that without the payload --- 10 minutes minimum.
-    \item The drone must be able to fly in fair conditions (assuming no rain, heavy winds, etc.).
-    \item The drone should be capable of auto-leveling, drift-correction, and perform emergency landing.
-    \item The drone should have systems for low-battery alarm and emergency protocol for low-battery and loss of control radio.
-\end{enumerate}
-
-\subsection{System Reliability and Durability}
-\begin{enumerate}[{NF.SR}.1:]
-    \item Brushless DC Motors used on the drone should have a life span of 10,000 flight-hours.
-    \item Non-computation-platform electronics are expected to have a life span of 20,000 flight hours if the procedures are followed correctly
-    \item The lithium polymer battery for the drone is expected to retain 75\% of its charge after 200 charge/discharge cycles. (TODO: make citation \url{http://learningrc.com/lipo-battery/}) 
+\begin{enumerate}[label=NF.DR.\arabic*, wide=1cm, widest=3cm, leftmargin=*, font=\bfseries, noitemsep,topsep=0pt, parsep=4pt, partopsep=0pt]
+	\item The multirotor RPAS shall be able to carry a maximum payload mass of 2.0 kg or 4.4 lb and achieve flight.
+	\item The multirotor RPAS maximum take off mass including the payload shall be less than 25.0 kg in compliance with Transport Canada TC-15263.\cite{tp15263}
+	\item The multirotor RPAS shall withstand 5 nautical miles (knots) of crosswind during operation.
+	\item The multirotor RPAS shall receive pilot TX signals via 2.4 GHz radio from 300 m or 1000 ft away.
+	\item The multirotor RPAS orientation LEDs shall be clearly visible from 300 m or 1000 ft away.
+	\item The multirotor RPAS flight controllers systems shall respond to pilot controls with less than 100ms of latency.
+	\item The multirotor RPAS shall have a hovering flight duration of 30 minutes with 1.0 kg of payload.
+	\item The brushless DC (BLDC) motors shall have a kV constant between 800 and 1200.
+	\item The BLDC motors and propellers shall output at least 8,000 N of thrust at maximum throttle.
+	\item The BLDC motors and propellers shall have an optimal efficiency of at least 50 N/W.
```

Comment:
I don't think this is necessary

Reply:
It is

Reply:
Let me elaborate, the datasheets I've found where I did my rough calculations from has a motor thrust-to power efficiency in the range of 4.0 to 6.0. 

The faster the motor spins, the faster the propeller is cutting through air, the higher the air resistance (drag is a function of speed^2). So an increase in motor power consumption has a diminishing return on force output.

A motor on full-power can probably output 7000N of thrust (714g if we divide 9.81m/s^2) at a efficiency below 4.0 g/W. Where a motor on half-power could output 3500N (357g) of thrust at a efficiency of about 5.0-6.0 g/W. At very low-power, the motor will only be able to lift a little, but the power is so small the ratio/efficiency is high.

Ideally, the drone should __hover__ (not climb, or with max. take-off payload) where the motors are operating at their optimal efficiency. Which is around 5-6g/W or at least 50 N/W. The reason I chose "hover" is because that is the __typical__ mode of operation.

Reply:
Maybe I should clarify my complaint :)

Yes, we need to lay out these values, but I'm not sure if the requirements document is the place to do it. Mieszko doesn't care about the kV constants of the BLDC motors, and I'd imagine these values are fairly specific to the drone implementation we choose.

Reply:
That's a good point. I think for now we'll leave it in. We should really start working on the other documents. If such "internal requirements" make is in the other docs, I guess we can remove it from here.

But for now, I think requirements are still requirements, even if the requirement is invisible to the end user